Gemfile

```diff
@@ -123,7 +123,7 @@ group :test do
   gem 'fakeweb', '~> 1.3.0', require: false
   gem 'minitest', require: false
   gem 'simplecov', require: false
-  gem "test-prof"
+  gem 'test-prof', git: 'https://github.com/danielwaterworth/test-prof.git'
```

Comment:
This is a bit of a blocker for me, can you push a temporary gem then until all the PRs are merged? or just carry a monkey patch locally till a new test-prof is out? 

In the past we have had issues with git dependencies in our Gemfile and even though this is only for test I am uneasy at the moment the only exceptions we allow are imports and rails master testing.